tests/utilities/streaming_data_measurment/StreamingMeasurement.serviceconfig

```diff
@@ -0,0 +1,18 @@
+{
+  "services": [
+    {
+      "displayName": "Sample Streaming Measurement (Py)",
+      "serviceClass": "ni.examples.SampleStreamingMeasurement_Python",
+      "descriptionUrl": "",
+      "providedInterfaces": [
+        "ni.measurementlink.measurement.v2.MeasurementService"
+      ],
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "MeasurementLink example that generates a predictable stream of data.",
+        "ni/service.collection": "NI.Examples",
```

Comment:
Consider changing "example" to "test" (or "test service" / "test measurement" where it makes sense).

Reply:
Updated